docs/guides/managing_packages.md

```diff
@@ -0,0 +1,254 @@
+During the development of agents and services, users need to manage other packages, either developing them themselves or fetching already available ones. This guide will show you which are the relevant directories during AEA development and how to get, use and publish packages.
```

Comment:
```suggestion
During the development of agents and services, users need to manage other packages, either developing them themselves or fetching already available ones. This guide will show you which are the relevant directories during AEA development and how to get, use and publish packages.
```
we care about development of autonomous services! AEA development is a secondary concern